stacks-signer/src/cli.rs

```diff
@@ -138,16 +138,24 @@ pub struct SignArgs {
 }
 
 #[derive(Parser, Debug, Clone)]
-/// Arguments for the Run and Dkg commands
+/// Arguments for the Dkg command
 pub struct RunDkgArgs {
     /// Path to config file
-    #[arg(long, value_name = "FILE")]
+    #[arg(long, short, value_name = "FILE")]
     pub config: PathBuf,
     /// The reward cycle the signer is registered for and wants to peform DKG for
     #[arg(long, short)]
     pub reward_cycle: u64,
 }
 
+#[derive(Parser, Debug, Clone)]
+/// Arguments for the Run command
+pub struct RunSignerArgs {
+    /// Path to config file
+    #[arg(long, short, value_name = "FILE")]
+    pub config: PathBuf,
+}
+
 #[derive(Parser, Debug, Clone)]
 /// Arguments for the generate-files command
 pub struct GenerateFilesArgs {
```

Comment:
Actually do you think you could clean up the GenerateFilesArgs generate files command? It should not need the reward cycle anymore. Actually might not need some other stuff either (we no longer generate stackerdb. so mind quickly going over as part of this PR? )

Reply:
yep np!

Reply:
LOL I legit missed the little arrows below this and thought I saw "reward_cycle" as part of generate files args...Ignore me.